knowledge-base/org.eclipse.scava.knowledgebase/Jenkinsfile

```diff
@@ -8,7 +11,7 @@ pipeline {
             sh "cp $MY_KB_APP_PROP src/test/resources/application.properties"
             sh 'mvn validate'
             sh 'mvn compile'
-            sh 'mvn package -Dmaven.test.skip=true'
+            sh 'mvn package'
```

Comment:
I suggest using 'mvn package -Dmaven.test.skip=true' instead of 'mvn package'. The tests will be executed on the test stage.

Reply:
The others changes should work.
Many concurrent commits on different branches have been properly managed.